server.js

```diff
@@ -0,0 +1,221 @@
+const http = require('http');
+const fs = require('fs');
+const PORT = 8080;
+const querystring = require('querystring');
+let date = new Date().toUTCString();
+const admin = {
+    bri: 'password',
+}
+
+const server = http.createServer((req, res) => {
+    // console.log(req);
+    // console.log('reqmethod', req.method);
+    // console.log('requrl', req.url);
+    let body = '';
+    req.on('data', (chunk) => {
+        body += chunk;
+    });
+    req.on('error', (error) => {
+        console.error(error);
+    })
+    req.on('end', () => {
+        let urlType;
+        let URL = req.url;
+        // GET METHOD
+        if (req.method === 'GET') {
+            if (req.url === '/' || req.url === '') {
+                urlType = 'html';
+                URL = '/index.html';
+            } else {
+                urlType = req.url.split('.');
+                urlType = urlType[1];
+            }
+            fs.readdir(`./public/`, (err, files) => {
```

Comment:
Error on fs.readdir is not being handled.

Reply:
To cut down on code, you don't need to do a fs.readdir then a fs.readFile. You can just do fs.readFile, and if there's an error, just send the 404.html file.